packages/core/parcel-bundler/src/Bundler.js

```diff
@@ -496,6 +497,8 @@ class Bundler extends EventEmitter {
   }
 
   async installDep(asset, dep) {
+    // Check if module is a URL, skip trying to resolve
+    if (isURL(dep.name)) return;
```

Comment:
Does it work if you do this check here: https://github.com/parcel-bundler/parcel/blob/master/packages/core/parcel-bundler/src/visitors/dependencies.js#L56 ?

Reply:
Not sure what to check for there... the only value that's the absolute URL at that line seems to be `node.callee.object.arguments[0].value`, but `if (isURL(node.callee.object.arguments[0].value)) return;` causes a failure the next time that line executes:

> $ node ./parcel-bundler/bin/cli.js index.html
Server running at http://localhost:1234 
🚨  /home/dandv/prg/parcel/packages/core/index.js: Cannot read property 'arguments' of undefined
    at CallExpression (/home/dandv/prg/parcel/packages/core/parcel-bundler/src/visitors/dependencies.js:56:34)
    at c (/home/dandv/prg/parcel/node_modules/babylon-walk/lib/index.js:149:9)
    at c (/home/dandv/prg/parcel/node_modules/babylon-walk/lib/index.js:186:9)
    at c (/home/dandv/prg/parcel/node_modules/babylon-walk/lib/index.js:186:9)
    at c (/home/dandv/prg/parcel/node_modules/babylon-walk/lib/index.js:186:9)
    at c (/home/dandv/prg/parcel/node_modules/babylon-walk/lib/index.js:183:11)
    at c (/home/dandv/prg/parcel/node_modules/babylon-walk/lib/index.js:186:9)
    at Object.ancestor (/home/dandv/prg/parcel/node_modules/babylon-walk/lib/index.js:210:5)
    at JSAsset.collectDependencies (/home/dandv/prg/parcel/packages/core/parcel-bundler/src/assets/JSAsset.js:82:10)
    at JSAsset.getDependencies (/home/dandv/prg/parcel/packages/core/parcel-bundler/src/Asset.js:79:18)

Reply:
#2380 should work, Note that this implementation doesn't work in internet explorer and any other browser that does not support `import()` natively.

In the future it would be a good thing to process these through parcel's bundle loader as well